platform/docs/versioned_docs/version-3.11/migration-guide/3p11-to-3p12/index.md

```diff
@@ -6,3 +6,11 @@ sidebar_label: 3.11 -> 3.12 beta
 # Migration Guide
 
 This guide provides information about migrating from OHIF version 3.11 to version 3.12 beta
+
+## Optional: Migrate modes to extend `modes/basic`
```

Comment:
no need to change this versioned docs, this should be in migration docs of 3.12 beta

Reply:
This is the 3.12 beta migration document - look at the file name:  3p11-to-3p12/index.md   If that isn't the right location for this document, that is a separate issue, and I can move this to version-3.12 as an entire directory.